zp-relayer/validation/validation.ts

```diff
@@ -77,39 +79,21 @@ const AjvSendTransactionSchema: JSONSchemaType<PoolTx> = {
       type: 'string',
       enum: [TxType.DEPOSIT, TxType.PERMITTABLE_DEPOSIT, TxType.TRANSFER, TxType.WITHDRAWAL],
     },
-    depositSignature: { type: 'string', nullable: true },
+    depositSignature: AjvNullableString,
   },
   required: ['proof', 'memo', 'txType'],
 }
 
-const AjvSendTransactionsSchema: JSONSchemaType<PoolTx[]> = {
-  type: 'array',
-  items: AjvSendTransactionSchema,
-}
-
-const AjvGetTransactionsSchema: JSONSchemaType<{
-  limit: number
-  offset: number
-  optimistic: boolean
-}> = {
+const AjvSendTransactionsSchema: JSONSchemaType<{ transactions: PoolTx[] }> = {
```

Comment:
Should we really change the schema here?

Reply:
You mean that if don't use trace id as request body property any more then we can restore previously used function signature?

Reply:
Yeah

Reply:
Agree, will restore previous signature